english/crouch-dragon/crouch-dragon.md

```diff
@@ -0,0 +1 @@
+Once upon a time, there is a baby dragon living in the sea. The baby dragon is very smart and nice, but some devils curse it. It lost its father, entwined into a family turnmoil, and then lost its beloved elder brother. It seemed to be doomed to live a miseral life. Nonetheless, with great perserverance, it rebounced back after crouching for a while. After having been surviving all the disasters and unfortune, it grew up with formidable willpower and undefeatable inner spirit. Here comes it, up front in the face of so-called destiny.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.